src/main/java/seedu/opus/sync/SyncServiceGtask.java

```diff
@@ -228,11 +249,27 @@ private TaskList createOpusTaskList() throws SyncException {
         googleAdaptedTask.setUpdated(new DateTime(updated));
 
         googleAdaptedTask.setStatus(source.getStatus().equals(Flag.COMPLETE)
-                                    ? TASK_STATUS_COMPLETE
-                                    : TASK_STATUS_INCOMPLETE);
+                ? TASK_STATUS_COMPLETE
+                        : TASK_STATUS_INCOMPLETE);
         return googleAdaptedTask;
     }
 
+    /**
+     * Deauthorize Opus by deleting the storedCredential file
+     */
+    public void deauthorize() {
```

Comment:
Well I though that `deauthorize` is more of a verb while `unauthorize` is more of an adjective.